DataFormats/Luminosity/interface/PCC.h

```diff
@@ -0,0 +1,74 @@
+#ifndef PCC_h
+#define PCC_h
+/** \class reco::PCC
+ *  
+ * Reconstructed PCC object that will contain the moduleID, BX, and counts.
+ *
+ * \authors: Sam Higginbotham shiggib@cern.ch and Chris Palmer capalmer@cern.ch
+ * 
+ *
+ */
+#include <algorithm>
+#include <string>
+#include <sstream>
+#include <iostream>
+#include <vector>
+
+namespace reco {
+class PCC {
+static constexpr unsigned int nEmBX=3564;//Empty BX to fill with counts
+    public:
+        PCC() : m_events(nEmBX){}
+        ////////////////////////////////////////////
+        void Increment(int mD,int BXid,int count){
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.